data/brands/shop/furniture.json

```diff
@@ -964,6 +964,7 @@
       "displayName": "Seats and Sofas",
       "id": "seatsandsofas-29dff1",
       "locationSet": {"include": ["de"]},
+	  "matchNames": ["Seats & Sofas"],
```

Comment:
Also the matcher can expand `&` to `and`, so "Seats & Sofas" is redundant and will be removed too.

There is a bunch of magic in the matching code - I tried to document it here:
https://github.com/osmlab/name-suggestion-index/wiki/Item-Property-Reference#matchnames-matchtags